### PR TITLE
[4.2] partial revert of #38638

### PR DESCRIPTION
**Pull request by brianteeman**
**I was wrong and should have listened more to what @hleithner was saying.** He was obviously correct that you can't use &hellip, in a language string in the cli.

~I still think the lack of readability of using utf characters in a search string is an issue so I have reverted the use to ...~

I have also reviewed all the other changes that I made in that PR and instead of reverting I am removing the &hellip; as it really served no purpose or was being misused. 

~As far as I can see now the remaining uses from that PR all relate to Read more where I still believe it is best to use &hellip but I would be happy to revert those as well.~

As far as I can tell the only remaining instance of ... is in the truncate functions where I believe it is correct

Sorry for causing this issue.

I am not sure if we are in a freeze for strings in 4.2 so if required I can rebase to 4.3

### CODE REVIEW

**Comment by HLeithner** [comment]
Thanks for creating this pr, do you really think that "please help" without the … is better?

I personal would be more happy remove all html encoded entries where possible.

**Comment by brianteeman** [comment]
> Thanks for creating this pr, do you really think that "please help" without the … is better?

did you mean "please wait ..."
If you did then yes I dont see it as needed. We're not using it to indicate there is more text that has been truncated but to indicate that something is happening. Which it doesnt actually do as its a static text

> 
> I personal would be more happy remove all html encoded entries where possible.

The theory I was trying to adopt with this change was that &hellip is good but lets keep it just for indicating truncation and limited to things that will only be consumed in html eg "read more ..."

but as it was my f - up. I am happy to simply revert completely

**Comment by HLeithner** [comment]
consistent would be better, so if you would be so kind remove all html encodings. and we don't use it for "there will something happen"

**Comment by brianteeman** [comment]
ok - will do

**Comment by brianteeman** [comment]
@hleithner updated the original post with current status (will update the styleguide when this is merged)

Note: there are a lot of instance of &quot &rarr etc

**Comment by HLeithner** [comment]
replacing &qoute blindly could be a problem because I wouldn't swear that the cms escapes it everywhere correctly, 
&rarr should be no problem to replace it with → same for &larr with ← if we have any …

**Comment by brianteeman** [comment]
yes I wouldnt want to do it blindly and it will probably need to be replaced by \"

&rarr, &larr, &lt, &gt should all be safe to change.

Do you want me to do it in this PR or seperate pr for each  type

**Comment by HLeithner** [comment]
I think one pr would ok for all this changes can be tested at once I think

**Comment by brianteeman** [comment]
changed to draft while I update the other html characterss

**Comment by brianteeman** [comment]
@HLeithner I'm not sure about the &lt, &gt changes. I cant seem to find a way to escape the <

**Comment by HLeithner** [comment]
hmm &lt and &gt is a problem, because if we don't use it we have valid html control characters, I would keep them. Also because we mix real html in the language strings.

**Comment by chmst** [comment]
Looks good for me. @brianteeman colud you remove the state draft?

@HLeithner I think it is better to get merge this one for now - if necessary we can improve it then. Or do you see problems?

File: administrator/language/en-GB/plg_quickicon_extensionupdate.ini [hunk 1]
```diff
@@ -4,8 +4,8 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_QUICKICON_EXTENSIONUPDATE="Quick Icon - Joomla! Extensions Updates Notification"
-PLG_QUICKICON_EXTENSIONUPDATE_CHECKING="Checking extensions &hellip;"
-PLG_QUICKICON_EXTENSIONUPDATE_ERROR="Unknown extensions &hellip;"
+PLG_QUICKICON_EXTENSIONUPDATE_CHECKING="Checking extensions"
+PLG_QUICKICON_EXTENSIONUPDATE_ERROR="Unknown extensions"
```
Comment (Quy) [inline administrator/language/en-GB/plg_quickicon_extensionupdate.ini]:
```suggestion
PLG_QUICKICON_EXTENSIONUPDATE_CHECKING="Checking extensions."
PLG_QUICKICON_EXTENSIONUPDATE_ERROR="Unknown extensions."
```

File: administrator/language/en-GB/plg_quickicon_overridecheck.ini [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_QUICKICON_OVERRIDECHECK="Quick Icon - Joomla! Overrides Update Notification"
-PLG_QUICKICON_OVERRIDECHECK_CHECKING="Checking overrides &hellip;"
+PLG_QUICKICON_OVERRIDECHECK_CHECKING="Checking overrides"
```
Comment (Quy) [inline administrator/language/en-GB/plg_quickicon_overridecheck.ini]:
```suggestion
PLG_QUICKICON_OVERRIDECHECK_CHECKING="Checking overrides."
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 53411abe538f61394a6bb2980ca70b37eb4f578c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39528">issues.joomla.org/tracker/joomla-cms/39528</a>.</sub>

**Comment by coolcat-creations** [comment]
How to test this?

**Comment by brianteeman** [comment]
go to any of the changed strings in the CMS and check that you are seeing the correct characters eg when it said &rarr you have a right arrow

**Comment by coolcat-creations** [comment]
Ok but before applying the patch I see an arrow and after also, Is this a problem in specific browsers?

**Comment by brianteeman** [comment]
that is a success. 

the problem with the change from arrows etc to html characters which this PR reverts was that it did not work in the cli

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 4.3-dev.

**Pull request closed without merging**